systers_portal/systers_portal/urls.py

```diff
@@ -21,6 +21,7 @@
     url(r'^community/', include('community.urls')),
     url(r'^community/', include('membership.urls')),
     url(r'^users/', include('users.urls')),
+    url(r'^meetup/', include('meetup.urls')),
```

Comment:
Alphabetically `meetup` is smaller than `users`. It should be placed before `users`.